doc/source/whatsnew/v0.18.2.txt

```diff
@@ -101,6 +101,7 @@ API changes
 ~~~~~~~~~~~
 
 
+- ``DataFrame.to_csv()`` has dropped the ``engine`` parameter (:issue:`13418`)
```

Comment:
It made it easier to see what I was doing when I was fixing the bug in `to_csv` though (didn't have to consider the other `engine`).